src/docker/Dockerfile

```diff
@@ -132,7 +132,7 @@ USER anvil
 WORKDIR /home/anvil
 
 # Install asdf. Get versions from https://github.com/asdf-vm/asdf/releases
-ARG ASDF_VERSION="0.7.8"
+ARG ASDF_VERSION="0.8.0"
```

Comment:
Question I've wonder about this:

Do we want to pull all of these binary versions out into one central file ? I noticed that they are scattered around the `Dockerfile` a bit.

Reply:
They live next to where they are needed so the docker layer cache gets busted less often. If they were all in one file changing one version would bust the whole cache even if it were all the way at the bottom of the dockerfile

Reply:
Yeah I had a feeling that was the reason.  Thanks for confirming.